app/src/main/java/com/andrew/tasky/agenda/presentation/adapters/AttendeeItemAdapter.kt

```diff
@@ -42,10 +42,10 @@ class AttendeeItemAdapter(
             deleteAttendeeButton.isVisible = !isCreatorHolder && !isAttendee
             creatorTextView.isVisible = isCreatorHolder
 
-            attendeeFullNameTextView.text = currentList[position].name
+            attendeeFullNameTextView.text = currentList[position].fullName
```

Comment:
You need this all the time, so better store the result of `currentList[position]` in a variable (and avoid weird bugs like we had before 😁 )